great_expectations/core/expectation_validation_result.py

```diff
@@ -636,11 +638,15 @@ def get_failed_validation_results(
         )
 
     def describe_dict(self) -> dict:
-        return {
+        data = {
             "success": self.success,
             "statistics": self.statistics,
             "expectations": [expectation.describe_dict() for expectation in self.results],
         }
+        if self.result_url:
+            data["result_url"] = self.result_url
```

Comment:
Conditionally adding it to the describe payload since OSS users will always see nulls. Is that worth an inconsistent payload shape?